src/plexil-adapter/OwInterface.cpp

```diff
@@ -58,18 +48,35 @@ const double PanTiltTimeout = 5.0; // seconds, made up
 
 // Lander operation names.
 // In some cases, these must match those used in PLEXIL and/or ow_lander
-const string Op_GuardedMove       = "GuardedMove";
-const string Op_GuardedMoveAction = "GuardedMoveAction";
+const string Op_GuardedMove       = "Guarded_move";
```

Comment:
Why does this use a different convention that the other commands?!

Reply:
Good catch, a vestige from when there was a duplicate GuardedMove.